packages/ubuntu_desktop_installer/lib/widgets/highlighted_button.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+const _kHighlightBackground = Color(0xFF0e8420);
+const _kHighlightForeground = Colors.white;
+
+class HighlightedButton extends StatelessWidget {
```

Comment:
https://github.com/canonical/ubuntu-desktop-installer/commit/67f6241d88ef6f1842586ab9a5edc9ce6371e19f exposed the style for highlighted buttons, rendering this class unnecessary. You should now use a regular `OutlinedButton` with `style: Theme.of(context).highlightedButtonTheme.style`.

Reply:
And since https://github.com/canonical/ubuntu-desktop-installer/commit/ca251e60236838839e46506e4fa6beb79b651975, you should simply use an `ElevatedButton` and not worry about the style, Yaru will handle it for you.